ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/managed-identity/provision/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-managed-identity-provision-chain.yaml

```diff
@@ -4,11 +4,11 @@ chain:
   - ref: azure-provision-resourcegroup
   - ref: azure-provision-vnet
   - chain: azure-provision-bastionhost
-  - ref: azure-provision-service-principal-minimal-permission
-  - ref: azure-provision-bastionhost-managed-identity
   - chain: ipi-conf-azure
   - ref: ipi-conf-azure-sharednetwork
   - ref: ipi-conf-manual-creds
+  - ref: azure-provision-service-principal-minimal-permission
+  - ref: azure-provision-bastionhost-managed-identity
   - ref: ipi-conf-azure-oidc-creds-provision
   - ref: ipi-conf-azure-provisioned-resourcegroup
```

Comment:
If possible, let us move `ipi-conf-azure-provisioned-resourcegroup` a bit before `azure-provision-service-principal-minimal-permission` to be safer.

Reply:
Or ignore my comments, `ipi-conf-azure-provisioned-resourcegroup` depends on `ipi-conf-azure-oidc-creds-provision`, so no need any update.